src/custom/utils/blocknative.ts

```diff
@@ -10,23 +11,42 @@ interface SDKError {
   transaction?: string
 }
 
+if (!BLOCKNATIVE_API_KEY) {
+  console.warn('[blocknative] Missing BLOCKNATIVE_API_KEY')
+  Sentry.captureException(new Error('Blocknative API key not set'), {
+    tags: { errorType: 'blocknative' },
```

Comment:
@alfetopito I created this new error type, not sure whether you would like to change it or add anything else?

Reply:
There seems to be something wrong with the error reporting

![Screen Shot 2022-09-05 at 14 00 59](https://user-images.githubusercontent.com/43217/188455682-412172c4-a3d5-4fca-80be-8f779fa579f7.png)
![Screen Shot 2022-09-05 at 14 01 23](https://user-images.githubusercontent.com/43217/188455689-caa2f501-f43e-4dfd-84ab-7476d310d236.png)

I've sent you an invite to Sentry

Reply:
At least the tagging is working!

![Screen Shot 2022-09-05 at 14 15 10](https://user-images.githubusercontent.com/43217/188458336-3904ccf9-fcb0-4f10-9116-30b52adeb88d.png)